src/ocrd/mets_server.py

```diff
@@ -428,8 +430,7 @@ def create_process(mets_server_url: str, ws_dir_path: str, log_file: str) -> int
 
     @staticmethod
     def kill_process(mets_server_pid: int):
-        subprocess_run(args=["kill", "-s", "SIGINT", f"{mets_server_pid}"], shell=False, universal_newlines=True)
-        return
+        return os.kill(mets_server_pid, signal.SIGTERM)
```

Comment:
```suggestion
        os.kill(mets_server_pid, signal.SIGINT)
        sleep(3)
        os.kill(mets_server_pid, signal.SIGKILL)
```

Reply:
This will result in `ProcessLookupError` in the `SIGKILL` line if the `SIGINT` succeeded.

Reply:
The SIGINT may suffice after all. It is just to cover the rare cases which may be ignored for now.

Reply:
Yeap, that is what I meant.